tests/backend/test_bosonic_backend.py

```diff
@@ -493,7 +513,7 @@ def test_measurement(self, alpha, r):
             sf.ops.Catstate(alpha) | q[0]
             sf.ops.Squeezed(r) | q[1]
             sf.ops.MeasureX | q[0]
-            sf.ops.MeasureX | q[1]
+            sf.ops.MeasureHD | q[1]
```

Comment:
?

Reply:
I realized I should test heterodyne here too, instead of just homodyne.

Reply:
Just curious: any reason `MeasureHD` refers to heterodyne, and not homodyne (to me it's a bit confusing)? 🤔